tests/functional/analyze/test_analyze.py

```diff
@@ -155,100 +154,6 @@ def test_compiler_info_files(self):
                 self.fail("json.load should successfully parse the file %s"
                           % info_File)
 
-    def test_compiler_includes_file_is_loaded(self):
```

Comment:
why was this test removed?

Reply:
Because we eliminated the `--compiler-includes-file` flag.